plugins/org.wso2.developerstudio.humantaskeditor/src/org/wso2/developerstudio/humantaskeditor/wizards/HumanTaskExportWizard.java

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wso2.developerstudio.humantaskeditor.wizards;
+
+import java.io.File;
+import java.io.IOException;
+import java.lang.reflect.InvocationTargetException;
+import java.util.List;
+
+import org.eclipse.core.resources.IProject;
+import org.eclipse.core.runtime.IProgressMonitor;
+import org.eclipse.core.runtime.Path;
+import org.eclipse.jface.dialogs.MessageDialog;
+import org.eclipse.jface.operation.IRunnableWithProgress;
+import org.eclipse.jface.viewers.IStructuredSelection;
+import org.eclipse.jface.wizard.Wizard;
+import org.eclipse.ui.IExportWizard;
+import org.eclipse.ui.IWorkbench;
+import org.wso2.developerstudio.humantaskeditor.HumantaskEditorConstants;
+
+
+public class HumanTaskExportWizard extends Wizard implements IExportWizard{
+    private HumanTaskExportWizardPage mainPage;
+    private IStructuredSelection selection;
+    
+    public boolean performFinish() {
+        IRunnableWithProgress op = new IRunnableWithProgress() {
+            public void run(IProgressMonitor monitor) throws InvocationTargetException {
+                try {
+                    doFinish(monitor);
+                } catch (Exception e) {
+                    throw new InvocationTargetException(e);
+                } finally {
+                    monitor.done();
+                }
+            }
+        };
+        try {
+            getContainer().run(true, false, op);
+        } catch (InterruptedException e) {
+            return false;
+        } catch (InvocationTargetException e) {
+            Throwable realException = e.getTargetException();
+            if (realException.getMessage().trim().equalsIgnoreCase(""))
+                MessageDialog.openError(getShell(), "Error", "Error occured while deploying the data service");
+            else
+                MessageDialog.openError(getShell(), "Error", realException.getMessage());
+            return false;
+        }
+        MessageDialog.openInformation(getShell(), "HT Process", "HT archive Exported successfully.");
```

Comment:
Let's use consistent information display format. The message should be  "Human task archive is exported successfully"